ironfist/src/cpp/combat/creatures.h

```diff
@@ -87,6 +87,7 @@ enum CREATURES
 	CREATURE_BLACK_KNIGHT = 75,
 	CREATURE_CATOBLEBA = 76,
 	CREATURE_TREANT = 77,
+	CREATURE_CYBER_BEHEMOTH = 78,
```

Comment:
FYI the Cyber Behemoth is currently slated to have creature ID 83. See https://docs.google.com/spreadsheets/d/1KjnW4nwSQr7y-M70wqjzTMMhbRpsKQ-tZNlIZoa92SE/edit#gid=0

It's perfectly okay to have skipped IDs, technically speaking.